src/Components/MainTableView/TableView.tsx

```diff
@@ -17,7 +17,8 @@ type TableViewState = {
   tableContentData: Array<any>,
   tableInfoData: string,
   selectedTable: string,
-  errorMessage: string
+  errorMessage: string,
+  isFetchingTuples: boolean
```

Comment:
Perhaps change this to a more generic name such as isReady to account for other loading things that might need to happen.

Reply:
changed to isLoading